package.json

```
@@ -62,6 +62,7 @@
     ]
   },
   "dependencies": {
+    "@badrap/result": "^0.2.12",
```

since this is v0 we may not want the caret, since there could be breaking changes if the middle digit increments
```suggestion
    "@badrap/result": "~0.2.12",
```